src/ruby/LICENSE

```diff
@@ -0,0 +1,95 @@
+Copyright (c) 2013 SymPy Development Team
```

Comment:
Ah I see. Why not to just install the LICENSE file from the root directory in the gem? I want to reduce file duplication and unnecessary files.